tests/test_aliases.py

```diff
@@ -98,17 +98,6 @@ class Config(BaseConfig):
     assert instance.to_dict(by_alias=True) == {"x": 123}
 
 
-def test_no_serialize_by_alias_code_generation_flag():
```

Comment:
These tests are no longer valid. `to_dict` and `from_dict` allow for arbitrary kwargs.